pipeline/extract/traffic_intensity.py

```diff
@@ -0,0 +1,57 @@
+import sys
+sys.path.append("./")
+
+import argparse
+import json
+import os
+import pandas as pd
+import requests
+from time import sleep
+from timeit import default_timer as timer
+from tqdm import tqdm
+from utils.data import HEALTH_ATLAS_VALUE_COLS
+
+
+cli = argparse.ArgumentParser(description="Download population data for each community area.")
```

Comment:
Change `"Download population data for each community area."` to `"Download traffic intensity data for each community area."`